### ISSUE TITLE
Hide Skip to Main Content link on mobile displays

### ISSUE THREAD

**Issue by magjarvis** [ATLAS-Illinois/Illinois-Framework-Theme#1143]
On both the iPhone and the Android phone, the Skip to Main Content link does not appear. It gains focus and is announced with VoiceOver and TalkBack but the link text is not visible. The Skip link is often considered unnecessary for the Mobile screen, in which case it should remain invisible and should also not accept focus.

**Expected behavior**
Hide or remove focus from Skip to Main Content link on mobile displays.

**Comment by trubach** [comment]
I made this a sub issue of #1075 - We should see if replacing the skipto with the campus version will solve this issue automatically.

**Comment by hpleas** [comment]
@magjarvis Could you provide a few steps for how I would see this.  An example site and URL would help.

**Comment by hpleas** [comment]
Please see merge request https://github.com/web-illinois/illinois_framework_theme/pull/1163 for resolution against issue requirements.
